frappe/website/website_generator.py

```diff
@@ -27,7 +27,10 @@ def get_website_properties(self, key=None, default=None):
 
 	def autoname(self):
 		if not self.name and self.meta.autoname != "hash":
-			self.name = self.scrubbed_title()
+			if self.meta.autoname == "naming_series:":
+				self.name = cleanup_page_name(self.name)
```

Comment:
Naming series output is not supposed to be processed in this manner. Adding a custom autoname to your controller is a better idea.